### PR TITLE
[RLlib] Revert "Revert "updated pettingzoo wrappers, env versions, urls""

### PR DESCRIPTION
**Pull request by sven1977**
Reverts ray-project/ray#21338

### CODE REVIEW

**Comment by sven1977** [comment]
Sorry @benblack769 , had to revert yesterday's merge again. Quick fix to our py3.6 build (adjusting the pettingzoo version for py<3.7 in requirements) should do ...

File: python/requirements/ml/requirements_rllib.txt
Comment (gjoliver) [inline python/requirements/ml/requirements_rllib.txt:15]: hey sven, sorry I am still a bit confused, is there any point to install 1.12 for py3.6 at this point? given all the api updates in env/wrappers/pettingzoo_env.py?
does pettingzoo_env.py still work with 1.12? or should we just declare we don't support any versions under 1.14 now?

**Comment by sven1977** [comment]
Looking good now.
@gjoliver , I removed pettingzoo and supersuit requirements entirely for py<3.7 as you suggested. You are right, we will officially not support those for py3.6.

**Comment by sven1977** [comment]
@benblack769 , tests seem to be all passing now. Waiting for approval, then another merge-attempt (hopefully, the last :) )

**Comment by gjoliver** [comment]
one test error that may not be related. thanks man!

**Review by gjoliver — APPROVED** [review]

**Comment by benblack769** [comment]
@sven1977 Thanks for getting this merged in!

**Pull request merged**
